connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -167,3 +168,12 @@ message LocalRelation {
   repeated Expression.QualifiedAttribute attributes = 1;
   // TODO: support local data.
 }
+
+// Relation of type [[Sample]] that samples a fraction of the dataset.
```

Comment:
`[[..]]` syntax is for Scaladoc to reference a column. Is it the same as protobuf? If not, we should probably just use backquotes.

Reply:
Maybe should be done separately because there seem many instances to fix here.

Reply:
yes I was following existing practice. 

The proto itself evolves fast. We need a lot more documentation on protobuf once it becomes stable a bit.

Reply:
@HyukjinKwon Apache Beam's documentation for proto (which is from Google internal proto documentation style) is inspiring: https://github.com/apache/beam/blob/master/model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/beam_runner_api.proto. 

Eventually I think we will need do the same. Given that this proto is also an API layer, we gonna document on every of the field with semantic and examples. It will take time though once our proto becomes stable.